Backend/dev/migrations/dynamic-offer-driver-app/0275-adding-new-issue-flow-related-data.sql

```diff
@@ -0,0 +1,139 @@
+ALTER TABLE atlas_driver_offer_bpp.issue_report RENAME COLUMN driver_id TO person_id;
+
+CREATE TABLE atlas_driver_offer_bpp.issue_message (
```

Comment:
These tables are already there in driver, right?

Reply:
This is a new table.